docs/README.md

```diff
@@ -34,37 +34,33 @@ If you're a **project manager** or other **decision maker** that needs to choose
 
-In particular we go into detail on how to [**deploy to Vercel**](https://github.com/frontity/docs/tree/c2586e3a601df487adc1c62bc6d4da02c6bedb8d/docs/deployment/deploy-using-vercel.mdgit), our recommended hosting platform. Why do we recommend it? Because it's serverless, cheap, includes a CDN, and is really easy to set up.
+In particular we go into detail on how to [**deploy to Vercel**](deployment/deploy-using-vercel.md), our recommended hosting platform. Why do we recommend it? Because it's serverless, cheap, includes a CDN, and is really easy to set up.
 
 ## **Guides**
 
 This is where you'll find an ever-growing series of [**Guides**](guides/). Some will help you in working with Frontity. Others will give you ideas for projects and provide you with the help you need to guide you from first steps to fully working project.
 
 This is where you can really have fun with Frontity and work toward your stretch goals.
 
-In this section we also provide you with some foundational knowledge that will assist you when working with Frontity. So if you need a refresher, or if you're new to the topics, we have included introductions to JavaScript/ES6 and React in this section.
+In this section we also provide you with some foundational knowledge that will assist you when working with Frontity. So if you need a refresher, or if you're new to the topics, we have included introductions to [JavaScript/ES6](guides/javascript-basics.md) and [React](guides/react-basic.md) in this section.
 
-Our troubleshooting guide is also here, for when you encounter problems. If your problem is not solved here then don't forget that you can also ask it in our [**Community Forum**](https://community.frontity.org).
+The [troubleshooting guide](guides/troubleshooting.md) is also here, for when you encounter problems. If your problem is not solved here then don't forget that you can also ask the community in the [**Frontity Community Forum**](https://community.frontity.org).
 
 ## **Contributing**
 
-Frontity is an open source project and we welcome contributions from the community. The [**Contributing**](contributing/) section of the documentation will guide you whether you want to make a single contribution or whether you intend to become a fully fledged contributor to the Frontity project.
+Frontity is an open source project and welcomes contributions in all forms. The [**Contributing**](contributing/) section of the documentation will guide you whether you want to make a single contribution or whether you intend to become a fully fledged contributor to the Frontity project.
 
-We can't wait to see where you take Frontity in the future.
+We also encourage you to join the [Community Forum](https://community.frontity.org) to share your projects and ideas with fellow users and community members. This is the best place to build new connections and get community support.
```

Comment:
Don't you think we should be consistent in avoiding the terms "our" or "we"?
Maybe we can directly say here...

> Join the [Community Forum](https://community.frontity.org) to share your projects and ideas with fellow users and community members. This is the best place to build new connections and get community support.

Reply:
Sure, that's fine too :)